app-wordpress/__main__.py

```diff
@@ -1,4 +1,4 @@
-"""A pulumi program to deploy asterion wordpress on kubernetes via helm."""
+"""A pulumi program to deploy wordpress to kubernetes via helm on the asterion rpi server."""
```

Comment:
Our application deployments are not specific to `infra-rpi`, they can just as rightly be deployed to `infra-aws` so I think we should keep this documentation generic and just reference "via helm to asterion infra".